filer/private/sass/components/_image-info.scss

```diff
@@ -171,11 +171,10 @@
     .image-preview-circle {
         position: relative;
         z-index: 1;
-        width: 26px;
-        height: 26px;
+        width: 30px;
+        height: 30px;
         // makes sure that circle is positioned correctly and centered #684
-        margin-top: -15px;
-        margin-left: -15px;
+        margin: -15px;
```

Comment:
do you really need all of them set to -15? usually it doesn't really work as expected